### PR TITLE
Remove github action cache

### PR DESCRIPTION
**Pull request by heyams**
got the idea from https://github.com/gradle/gradle-build-action/issues/115

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -71,6 +71,8 @@ jobs:
           java-version: 11
       - name: Test
         uses: gradle/gradle-build-action@v2
+        env:
+          GRADLE_BUILD_ACTION_CACHE_KEY_PREFIX: ${{github.workflow}}#${{github.run_number}}:${{github.run_attempt}}-
```
Comment (trask) [inline .github/workflows/pr.yaml:75]: I don't think we will get any benefit from cache if the prefix changes on every run

**Review by trask — APPROVED** [review]

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (heyams) [reply .github/workflows/pr.yaml:75]: let's see how long it takes for each run. If it's no different from cache, i think it's ok not to cache? it's all about timing. 
if it takes long time, will revert back to a static string.

Comment (heyams) [reply .github/workflows/pr.yaml:75]: we can keep updating the prefix whenever running into the cache issue.

Comment (heyams) [reply .github/workflows/pr.yaml:75]: will merge it in for now.. azure pipeline build failed due to sporadic link failure. it's taking 30 mins or so, but regular build also takes around 30 mins.. will check against other runs.

**Pull request merged**